### ISSUE TITLE
Please improve Docker documentation

### ISSUE THREAD

**Issue by n0toose** [misskey-dev/misskey#7654]
## Summary

Hi, I'm currently in the process of setting up Misskey on Docker. The process is a literal nightmare and requires a decent amount of knowledge that I've hopefully obtained over the past 24 hours. Here's a few suggestions for the Docker documentation:

- How to get the IP addresses of each container and a stern reminder to update the environment file with the IP addresses OR that it should be replaced with the hostname
- After updating the IP addresses, run `docker-compose build web` so that the IPs won't change as soon as you build the containers again. (e.g. `docker-compose build`)
- Some sort of a pointer pointing out other options rather than just the Postgres stuff (if any) in `docker.env`, a hyperlink could help.

I don't have the time to do any of this myself as I'm still very much struggling.

**Comment by Candinya** [comment]
Using hostnames like `db` `redis` to identify the container is more recommended than forcing any IPs. Clearly described in docs, you can use **hostname**  (service's name by default) of containers described in `docker-compose.yml` to connect different containers to `web` container. 

![image](https://user-images.githubusercontent.com/20502130/130313463-17c324a5-75c3-40cb-bbf2-13c58d9869f4.png)

As for `docker.env`, currently it just points out the necessary envirinment variables for `db` container. You may refer to [Official documents of env-file for docker-compose](https://docs.docker.com/compose/env-file/). If other containers need env settings too, you can simply split them into different files.

For better understands of docker-compose I prefer the official document [Get started with Docker Compose](https://docs.docker.com/compose/gettingstarted/) . Like in the demo code, using **hostname** defined in `docker-compose.yml` is recommended.

![image](https://user-images.githubusercontent.com/20502130/130313676-9b00de7d-4e19-4ddd-a2b7-29cb4de0d9be.png)
![image](https://user-images.githubusercontent.com/20502130/130313681-95ab5bf9-5833-4b0d-8f35-9f384e615d35.png)

Thanks for your participation. If there's any further questions, feel free to let us know.

**Comment by n0toose** [comment]
Yeah, I finally got that part. Thanks a lot for your support, but I do feel like it'd be much better if some of that was clarified better in the documentation itself. Should I go ahead and try to condense that information (while adding the document you just mentioned as an extra "do your homework thing"), make it newb-friendly and open up a pull request?

**Comment by n0toose** [comment]
At least I got forced to learn a LOT of things about `docker-compose` and how it works, I think I've gained a much better understanding of it now, so, that was a nice unintended consequence of that?
